src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsTableViewCell.swift

```diff
@@ -13,34 +13,175 @@ class HomeStatisticsTableViewCell: UITableViewCell {
         super.awakeFromNib()
 
 		self.addGestureRecognizer(scrollView.panGestureRecognizer)
+
+		accessibilityIdentifier = AccessibilityIdentifiers.Statistics.General.tableViewCell
     }
 
-	// MARK: - Internal
+	override func layoutSubviews() {
+		super.layoutSubviews()
+		self.scrollView.bounds.origin.x = self.scrollView.frame.size.width
+	}
+
+	override func setEditing(_ editing: Bool, animated: Bool) {
+		super.setEditing(editing, animated: animated)
 
+		Self.editingStatistics = editing
+		
+		stackView.arrangedSubviews.forEach { view in
+			let card = view as? HomeStatisticsCardView
+			card?.setEditMode(editing, animated: animated)
+		}
+	}
+
+	// MARK: - Internal
+	
+	// swiftlint:disable:next function_parameter_count
 	func configure(
```

Comment:
General remarks: 
- VoiceOver is not yet working properly. It skips all the local incidence cells and doesn't scroll correctly. 
- Shouldn't the "Lokale 7-Tage-Inzidenz hinzufügen" Button be greyed out instead of removed? Looks like that in Figma and Android also had it that way this morning.